### PR TITLE
Fix/summit prte 2

### PR DESCRIPTION
**Pull request by andre-merzky**
This PR changes the way the PRTE DVM reports the HNP URI.  That was previously parsed out of the hnp process output - but that parsing yields an incorrect result in some debug modes.  The URI is now communicated via the FS, which is slower but more reliable.   The DVM setup now also uses a hostfile instead of relying on LSF env information for node selection.

### CODE REVIEW

**Comment by andre-merzky** [comment]
@iparask : please review - thanks

File: src/radical/pilot/agent/lm/prte.py [hunk 1]
```diff
@@ -81,41 +96,38 @@ def lrms_config_hook(cls, name, cfg, lrms, logger, profiler):
...
+                        if '://' in line:
+                            dvm_uri = line.strip()
+                            break
 
-            elif 'DVM ready' in line:
-                if not dvm_uri:
-                    raise Exception("VMURI not found!")
+            except Exception as e:
+                logger.debug('DVM check: %s' % e)
+                time.sleep(0.5)
```
Comment (mturilli) [inline src/radical/pilot/agent/lm/prte.py]: See my comment to this code on PR #1902

Comment (andre-merzky) [reply src/radical/pilot/agent/lm/prte.py]: Ack, same reply.

**Comment by andre-merzky** [comment]
> Can we get travis to check flake8 and pylint on this file? the rm_tests branch also has a test for this launch method. It would be nice to make sure that it still passes.

Sure - pylint and flake8 both pass on the prte LM.

Thanks both!

**Pull request merged**